spec/index.bs

```diff
@@ -5978,11 +5978,16 @@ dictionary GPUImageCopyExternalImage {
         </div>
 </dl>
 
-### Image Copies ### {#image-copies}
+### <dfn dfn lt="image copy">Image Copies</dfn> ### {#image-copies}
 
 WebGPU provides {{GPUCommandEncoder/copyBufferToTexture()}} for buffer-to-texture copies and
 {{GPUCommandEncoder/copyTextureToBuffer()}} for texture-to-buffer copies.
 
+Pixels are tightly-packed in the linear-memory layout of each row of an image copy.
```

Comment:
Maybe worth explaining what this is, then adding a note saying "this is often referred to as linear memory layout"?
I.e. say that texel (i+1, j) follows (i, j), and row (_, j+1) follows row (_, j).

Reply:
I tried to sidestep this by only explaining the layout of each individual row, not the whole image, but we do need this somewhere eventually. GPUImageDataLayout has TODOs to describe linear memory layouts.

Reply:
Moved this to that section (which makes more sense) and rewrote it in terms of texel blocks.